working/augmentation-libraries/feature-specification.md

```diff
@@ -142,13 +142,32 @@ source order, and then recursing on `part`-directives in source order.
 
 ## Augmentation declarations
 
-Augmentation declarations are declarations marked with the new built-in
-identifier `augment`, which makes the declaration augment an existing
-declaration, which is itself a normal declaration with zero or more prior
-augmentations applied. An augmentation declaration does not  introduce a new
-name into the surrounding scope, it attaches itself to the existing name.
+Every declaration in Dart without this feature is _not_ an augmentation
+declaration. We say that it is an _introductory_ declaration.
 
-Making `augment`  a built-in identifier is language versioned, to make it
+The _context_ of a top-level declaration in a Dart file is the library of the
```

Comment:
This is a lot of text, which I find a lot more difficult to comprehend than the previous text. Maybe that is unavoidable to get a level of precision that is necessary? I don't have a concrete suggestion in any case...

Reply:
I added a couple of commentary paragraphs, I think that amounts to a substantial improvement.